DEVELOPMENT.md

```diff
@@ -4,6 +4,13 @@
 > to find out about how to contribute to Flux and how to interact with the
 > Flux Development team.
 
+## Installing required dependencies
+There are a number of dependencies required to be able to run image-reflector-controller and its test-suite locally. 
+* [Install Go](https://golang.org/doc/install)
+* [Install Kustomize](https://kubectl.docs.kubernetes.io/installation/kustomize/)
+* [Install Docker](https://docs.docker.com/engine/install/)
+* [Install Kubebuilder](https://book.kubebuilder.io/quick-start.html)
```

Comment:
I don't think installing Kubebuilder is mandatory. It's probably useful only for initial scaffolding. Maybe have it as optional?